aws-rds-dbclusterparametergroup/src/main/java/software/amazon/rds/dbclusterparametergroup/DefaultHandlerConfig.java

```diff
@@ -0,0 +1,16 @@
+package software.amazon.rds.dbclusterparametergroup;
+
+import java.time.Duration;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+
+@SuperBuilder
+public class DefaultHandlerConfig extends software.amazon.rds.common.handler.HandlerConfig {
```

Comment:
nit: `DefaultHandlerConfig` implies that there are non-default handler configs. This class seems like a config for `DBClusterParameterGroup` handler.

Reply:
Proposed naming:
- `DBClusterParameterGroupHandlerConfig` (clear but unreasonably long)
- `HandlerConfigExt` (unclear, but short and efective)
- `LocalHandlerConfig` (optimal length-wise, but a bit unclear how far the zone of the locality spreads, though)